### PR TITLE
Pull in ethash and go-isatty updates

### PR DESCRIPTION
**Pull request by mrdomino**
Fixes build on OpenBSD. See also #1735.

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Sun Aug 30 17:41:51 UTC 2015_

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -51,7 +50,7 @@
 		},
 		{
 			"ImportPath": "github.com/mattn/go-isatty",
-			"Rev": "fdbe02a1b44e75977b2690062b83cf507d70c013"
+			"Rev": "ae0b1f8f8004be68d791a576e3d8e7648ab41449"
```
Comment (karalabe) [inline Godeps/Godeps.json]: Could you please update to latest go-isatty (7fcbc72f853b92b5720db4a6b8482be612daef24)?

Comment (fjl) [inline Godeps/Godeps.json]: no need

Comment (mrdomino) [inline Godeps/Godeps.json]: Done regardless -- can't hurt.

**Pull request merged**